uaa/src/main/webapp/WEB-INF/spring/saml-providers.xml

```diff
@@ -144,6 +149,8 @@
         <bean id="samlWebSSOProcessingFilter" class="org.springframework.security.saml.SAMLProcessingFilter">
             <property name="authenticationManager" ref="authenticationManager" />
             <property name="authenticationSuccessHandler" ref="successRedirectHandler" />
+            <property name="authenticationFailureHandler" ref="samlLoginFailureHandler" />
+            <property name="contextProvider" ref="contextProvider" />
```

Comment:
@amiri-ge - not sarcastic, one shouldn't have to view source code of an external library to see what the hell is going on :)